wandb/sdk/wandb_init.py

```diff
@@ -1026,6 +1025,8 @@ async def display_init_message() -> None:
         except TimeoutError:
             pass
 
+        backend.interface.deliver_probe_system_info()
```

Comment:
Since we don't wait for a response (and especially since wandb-core never responds), this should use the `publish` verb instead and not make a mailbox handle.